app/src/main/java/com/example/movies/BindingAdapter.kt

```diff
@@ -47,3 +57,9 @@ fun bindStatus(statusImageView: ImageView, status: MoviesApiStatus?) {
         }
     }
 }
+
+@BindingAdapter("knowFor")
+fun knowFor(textView: TextView, data: List<Movie>?) {
+    val titleMovies = data?.map { it.title }
+    textView.text = "Know for: ${titleMovies?.toString()?.replace("[", "")?.replace("]", "")}"
```

Comment:
Maybe a better way to do this is change this line for this one `titleMovies?.forEach { it -> if (it != null) textView.text = textView.text.toString() + "$it, " }`